src/components/common/icons/LineArrow.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+import { ReactComponent as LineArrowLeftSvg } from '../../../assets/images/icons/arrow/line-left.svg';
+import { ReactComponent as LineArrowRightSvg } from '../../../assets/images/icons/arrow/line-right.svg';
+
+type Direction = 'left' | 'right';
+
+export default function LineArrow({ direction }: { direction: Direction }) {
+  return <>{direction === 'left' ? <LineArrowLeft /> : <LineArrowRight />}</>;
+}
```

Comment:
구조분해 할당에 대응하기 위해 ({ }:{ }) 모양으로 타입을 지정하는 것보다는 ({ }:Interface) 처럼 인터페이스로 처리하는게 좋아보입니다

Reply:
그 방법이 깔끔하긴 하겠네요!
type을 이렇게 써보는 건 어떨까요??!

```suggestion
type Direction = {
  direction: 'left' | 'right'
}

export default function LineArrow({ direction }: Direction) {
  return <>{direction === 'left' ? <LineArrowLeft /> : <LineArrowRight />}</>;
}
```

얼마 전에 type과 interface에 대해서 정리해봤는데요,
지금은 type과 interface가 거의 같은 기능을 제공하긴 하지만 조금 더 의미에 맞는 것을 사용하고 싶어서 파고들게 되었네요 ㅎㅎ

https://hongbi.notion.site/Interface-vs-Type-299d23cca70f4a1996f3a8bf88d0b506